test/integration/test-amp-bind.js

```diff
@@ -71,7 +71,7 @@ describe.configure().ifNewChrome().run('amp-bind', function() {
     return fixture.awaitEvent(BindEvents.RESCAN_TEMPLATE, ++numTemplated);
   }
 
-  describe('with [text] and [class]', () => {
+  describe.configure().run('with [text] and [class]', () => {
```

Comment:
Are the changes in this file necessary? The top-level describe in this file already has `configure()`.